### PR TITLE
Add notice on swerve support

### PR DESCRIPTION
**Pull request by Daltz333**

### CODE REVIEW

File: source/docs/software/pathplanning/index.rst [hunk 1]
```diff
@@ -3,6 +3,16 @@ Path Planning
 
 Path Planning is the process of creating and following trajectories. These paths use the WPILib trajectory APIs for generation and a :ref:`Ramsete Controller <docs/software/advanced-controls/trajectories/ramsete:Ramsete Controller>` for following. This section highlights the process of characterizing your robot for system identification, trajectory following and usage of PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for additional information about the API and non-commandbased usage.
 
+Notice on Swerve Support
+------------------------
+
+Swerve support in path following has a couple of hiccups that teams need ot be aware of:
+- WPILib currently does not support swerve in simulation, please see `this <https://github.com/wpilibsuite/allwpilib/pull/3374>`__ pull request.
+- SysID only supports tuning the swerve heading using a simple motor project and is unable to collect module velocity data.
```
Comment (mdurrani808) [inline source/docs/software/pathplanning/index.rst:11]: "SysID only supports tuning the swerve heading"
Is heading the best word to use here? I assume you mean the swerve azimuth or tuning the swerve turning motor. Also, is a simple motor project still a thing in SysID? To my knowledge, it is just general mechanism now.

**Comment by jasondaming** [comment]
I think we should add the two workarounds:
1) You can chacterize velocity by locking the module rotation
2) You can still use PathWeaver and just import a Waypoints file rather than the json and generate the trajectory.  You would also need to ignore the interconnecting lines between waypoints in PW.

**Comment by Daltz333** [comment]
1. How should teams lock module rotation?
2. This should probably get added to the Pathweaver docs somewhere. Can you provide an example or a more detailed description of this?

**Comment by jasondaming** [comment]
1) This is pretty module dependent.  Easiest thing is probably the block of wood seen [here](https://www.chiefdelphi.com/t/sds-mk4-swerve-module/394644/113?u=jdaming), but I won't be able to actually try this for another week or so.

2) I have been exploring this a lot with Tyler.  Doing it with multiple different robot headings different from the direction of travel is not possible with our current trajectory generation code.  I am working on an example that shows modifications allowing this to be done.  My suggestion would be to include of the single heading approach now and a link to my repo (will update with it "soon") for the more complicated.

**Review by jasondaming — APPROVED** [review]

File: source/docs/software/pathplanning/index.rst [hunk 2]
```diff
@@ -3,6 +3,16 @@ Path Planning
 
 Path Planning is the process of creating and following trajectories. These paths use the WPILib trajectory APIs for generation and a :ref:`Ramsete Controller <docs/software/advanced-controls/trajectories/ramsete:Ramsete Controller>` for following. This section highlights the process of characterizing your robot for system identification, trajectory following and usage of PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for additional information about the API and non-commandbased usage.
 
+Notice on Swerve Support
+------------------------
+
+Swerve support in path following has a couple of issues that teams need to be aware of:
```
Comment (sciencewhiz) [inline source/docs/software/pathplanning/index.rst:9]:
```suggestion
Swerve support in path following has a couple of limitations that teams need to be aware of:
```

File: source/docs/software/pathplanning/index.rst [hunk 3]
```diff
@@ -3,6 +3,16 @@ Path Planning
...
 
+Notice on Swerve Support
+------------------------
+
+Swerve support in path following has a couple of limitations that teams need to be aware of:
+- WPILib currently does not support swerve in simulation, please see `this <https://github.com/wpilibsuite/allwpilib/pull/3374>`__ pull request.
+- SysID only supports tuning the swerve heading using a General Mechanism project and does not regularly support module velocity data. A workaround is to lock the module's heading into place. This can be done via blocking module rotation using something like a block of wood.
+- Pathweaver and Trajectory following currently do not take heading into account. Path following using the WPILib trajectory framework on swerve will be the same as a DifferentialDrive robot.
+
+We are sorry for the inconvenience at this time.
```
Comment (sciencewhiz) [inline source/docs/software/pathplanning/index.rst:14]:
```suggestion
We are sorry for the inconvenience.
```

File: source/docs/software/pathplanning/index.rst [hunk 4]
```diff
@@ -3,6 +3,16 @@ Path Planning
...
 
+Notice on Swerve Support
+------------------------
+
+Swerve support in path following has a couple of limitations that teams need to be aware of:
+- WPILib currently does not support swerve in simulation, please see `this <https://github.com/wpilibsuite/allwpilib/pull/3374>`__ pull request.
+- SysID only supports tuning the swerve heading using a General Mechanism project and does not regularly support module velocity data. A workaround is to lock the module's heading into place. This can be done via blocking module rotation using something like a block of wood.
+- Pathweaver and Trajectory following currently do not take heading into account. Path following using the WPILib trajectory framework on swerve will be the same as a DifferentialDrive robot.
```
Comment (sciencewhiz) [inline source/docs/software/pathplanning/index.rst:12]:
```suggestion
- Pathweaver and Trajectory following currently do not incorporate independent heading. Path following using the WPILib trajectory framework on swerve will be the same as a DifferentialDrive robot.
```

**Comment by mdurrani808** [comment]
It would not hurt to add the thickness of wood (3/4in) needed to lock the modules..

**Review by sciencewhiz — APPROVED** [review]

**Comment by sciencewhiz** [comment]
Wouldn't the size and shape of wood vary based on what type of module?

**Pull request merged**